commands/config_e2e_test.go

```diff
@@ -111,3 +115,60 @@ func (s *MmctlE2ETestSuite) TestConfigSetCmd() {
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestConfigEditCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Edit a key in config", func(c client.Client) {
+		printer.Clean()
+
+		// check the value before editing
+		args := []string{"ServiceSettings.EnableSVGs"}
+		err := configGetCmdF(c, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().False(*printer.GetLines()[0].(*bool))
+		s.Require().Len(printer.GetErrorLines(), 0)
+		printer.Clean()
+
+		// create a shell script to edit config
+		content := `#! /bin/bash
+sed -i'old' 's/\"EnableSVGs\": false/\"EnableSVGs\": true/' $1
+rm $1'old'`
+
+		file, err := ioutil.TempFile(os.TempDir(), "config_edit_*.sh")
+		s.Require().Nil(err)
+		defer func() {
+			file.Close()
+			os.Remove(file.Name())
+			resetCmd := &cobra.Command{}
+			resetCmd.Flags().Bool("confirm", true, "")
+			s.Require().Nil(configResetCmdF(c, resetCmd, []string{"ServiceSettings"}))
+		}()
+		_, err = file.Write([]byte(content))
+		s.Require().Nil(err)
+		editorCmd := exec.Command("chmod", "+x", file.Name()) //nolint:gosec
```

Comment:
Is there a method where we don't need to bypass the security check?

Reply:
Will look into it. I wanted to make the shell script executable. I was getting `G204: Subprocess launched with function call as argument or cmd arguments` lint error, hence added this. Will try to see if I can find a better way to do this.

Reply:
Probably you can use [`os.Chmod`](https://golang.org/pkg/os/#Chmod) instead to add execution permissions, so the linter won't complain